pkg/agent/openflow/multicast.go

```diff
@@ -195,6 +213,18 @@ func (f *featureMulticast) replayGroups() {
 	}
 }
 
+func (f *featureMulticast) multicastForwardInboundFlow(groupID binding.GroupIDType) []binding.Flow {
```

Comment:
Do we need to support the scenario that multicast sender is a host process on the Node or a Pod with hostNetwork? That is to say, shall we add a flow to support in_port=hostInterface, action is output it to OVS multicast pipeline and the uplink ?

Reply:
Updated. I think it should be supported. I have added the flows as you specified.